src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -253,6 +253,7 @@ public void updateObservablePersonList(Predicate<Person> predicate) {
     @Override
     public void updateObservablePersonList() {
         filteredPersons.setPredicate(PREDICATE_SHOW_ALL_PERSONS);
+        observablePersons.setComparator(COMPARATOR_CONTACT_INDEX_PERSON.reversed());
```

Comment:
Reverses and unreverses the ordering so that UI components are responsive. Fixes #305